modules/openapi-generator-cli/pom.xml

```diff
@@ -78,6 +77,12 @@
             <groupId>org.openapitools</groupId>
             <artifactId>openapi-generator</artifactId>
             <version>${project.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
```

Comment:
@jimschubert:
> I've left the exclusion of slf4j-simple in the dependency declaration in openapi-generator-cli so we'd have record of this exclusion in the master branch.

For me this is similar to unnecessary or commented code, it will be very difficult to understand from the outside or for us in few weeks or months. I would remove those lines.

Reply:
If we remove this, it can result in a transient dependency pulling in the simple binding, and a warning.